tests/test_policy.py

```diff
@@ -14,3 +16,5 @@ def test_policy_checks_glibc():
     assert policy > POLICY_PRIORITY_LOWEST
     policy = versioned_symbols_policy({"some_library.so": {"GLIBC_999"}})
     assert policy == POLICY_PRIORITY_LOWEST
+    policy = versioned_symbols_policy({"some_library.so": {"OPENSSL_1_1_0"}})
+    assert policy == POLICY_PRIORITY_HIGHEST
```

Comment:
Since `OPENSSL_1_1_01 isn't in the whitelist, I might have expect this to give POLICY_PRIORITY_LOWEST, but perhaps there's something I'm not thinking of.